### DISCUSSION TITLE
Isn´t really a way to open index.html SPA over file:// ?

### DISCUSSION THREAD

**Discussion by rafael-mq** [quasarframework/quasar#9661] [CLI - PWA mode] [answered]
Hey guys.

Quasar newbie here. I am evaluating Quasar for a project that I have to access the built index.html over file protocol.
Yes, I noticed the warning message on `$ quasar build` output:
` Tip: Built files are meant to be served over an HTTP server
      Opening index.html over file:// won't work`

But I achieved my objective on a Vue CLI (vue create) project by setting publicPatch property in vue.config.js as follows:
`publicPath: process.env.NODE_ENV === 'production'
      ? './'
      : '/',`

Isn't there really any way to achieve this with Quasar?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@rafael-mq 

> Tip: Built files are meant to be served over an HTTP server Opening index.html over file:// won't work

So what happens when you do this in quasar( in the quasar.conf.js file)?:
`publicPath: process.env.NODE_ENV === 'production' ? './' : '/',`

See ( use Qv1 with app V2) :
https://quasar.dev/quasar-cli/quasar-conf-js#property-devserver

btw use router mode **hash** (not history).
`vueRouterMode: 'history'`

**Reply by rafael-mq** [reply]
Thanks a lot @ontwikkelfabriek 

As strange as it may look, after a few "try and fail" iterations, I succeeded by not setting the `publicPath` configuration. (having the default on).

To give a little more context: I have a mobile app with a webview that renders a webpage from its local file storage. So far I achieved it with vanilla html+js+css but I wanted to make use of Vue + Quasar.

Also I'll keep in mind your router mode tip.
